src/utils/general.util.js

```diff
@@ -0,0 +1,6 @@
+
+export default {
+  capitalize(string) {
```

Comment:
mislim da je ova funkcija visak, lodash ima to vec
https://lodash.com/docs/4.17.5#upperFirst